### PR TITLE
i18n(fr): Update `guides/backend/*` from #9240

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)

i18n(fr): Update `guides/backend/*` from #9240 : 

- [x] appwrite
- [x] google-firebase
- [x] ~~index~~ (moved in #9240)
- [x] neon
- [x] sentry
- [x] supabase
- [x] turso
- [x] xata

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/backend/google-firebase.mdx [hunk 1]
```diff
@@ -19,7 +21,7 @@ Consultez notre guide dédié au [déploiement sur Firebase hosting](/fr/guides/
 ### Prérequis
 
 - Un [projet Firebase avec une application Web configurée](https://firebase.google.com/docs/web/setup).
-- Un projet astro avec [server-side rendering (SSR)](/fr/guides/on-demand-rendering/) activé.
+- Un projet astro avec [`output: 'server'` pour un rendu à la demande](/fr/guides/on-demand-rendering/) activée.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/backend/google-firebase.mdx:24]: I forgot to add the nit... 😅 
```suggestion
- Un projet Astro avec [`output: 'server'` pour un rendu à la demande](/fr/guides/on-demand-rendering/) activée.
```

**Comment by thomasbnt** [comment]
> A nit, and you missed an `hybrid` > `static` in `supabase.mdx` (l215, I can't add a suggestion there).

Thanks, fixed with [`d5a2f9f` (#10180)](https://github.com/withastro/docs/pull/10180/commits/d5a2f9f28d78b4ea780bf921b9ae22ce7257ce16)

**Review by ArmandPhilippot — APPROVED** [review]
LGTM!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**